code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -42,7 +42,8 @@
 /obj/item/weapon/extinguisher/New()
 	..()
 	create_reagents(max_water)
-	reagents.add_reagent("water", max_water)
+	reagents.add_reagent("water", (max_water - 10))

```

Comment:
I could run a few tests, though I didn't want to make the amount too high since people will start freezing turfs and causing brain damage with extinguisher sprays